pydantic/_internal/_annotated_handlers.py

```diff
@@ -88,3 +88,19 @@ def generate_schema(self, __source_type: Any) -> core_schema.CoreSchema:
             CoreSchema: the `pydantic-core` CoreSchema generated.
         """
         raise NotImplementedError
+
+    def resolve_ref_schema(self, __maybe_ref_schema: core_schema.CoreSchema) -> core_schema.CoreSchema:
+        """Get the real schema for a `definition-ref` schema.
+        If the schema given is not a `definition-ref` schema, it will be returned as is.
+        This means you don't have to check before calling this function.
+
+        Args:
+            __maybe_ref_schema: A CoreSchema, ref based or not.
+
+        Raises:
+            LookupError: if the ref is not found.
+
+        Returns:
+            CoreSchema: A concrete CoreSchema.
```

Comment:
```suggestion
            A concrete `CoreSchema`.
```

Reply:
Specifying the return type is not required. mkdocstrings infers it from the signature.